kie-dmn-feel/pom.xml

```diff
@@ -71,6 +71,26 @@
 
   <build>
     <plugins>
+    
+        <plugin>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <configuration>
+            <fork>true</fork>
+            <meminitial>128m</meminitial>
+            <maxmem>512m</maxmem>
+            <showDeprecation>true</showDeprecation>
+            <showWarnings>true</showWarnings>
+            <source>1.8</source>
+            <target>1.8</target>
+            <testSource>1.8</testSource>
+            <testTarget>1.8</testTarget>
```

Comment:
OK, cool!

Just FYI, unless you need to remove some default arg., there is a way to just add only the single argument:

```
        <plugin>
          <artifactId>maven-compiler-plugin</artifactId>
          <configuration>
            <compilerArgs combine.children="append">
              <arg>-parameters</arg>
            </compilerArgs>
          </configuration>
        </plugin>
```
This is not very well known Maven feature, but comes handy sometimes.